README.md

```diff
@@ -1,9 +1,49 @@
 Mapper Attachments Type for Elasticsearch
 =========================================
 
-The mapper attachments plugin adds the `attachment` type to Elasticsearch using [Apache Tika](http://lucene.apache.org/tika/).
-The `attachment` type allows to index different "attachment" type field (encoded as `base64`), for example,
-microsoft office formats, open document formats, ePub, HTML, and so on (full list can be found [here](http://tika.apache.org/1.10/formats.html)).
+The mapper attachments plugin lets Elasticsearch index file attachments in over a thousand formats (such as PPT, XLS, PDF) using the Apache text extraction library [Tika](http://lucene.apache.org/tika/).
+
+In practice, the plugin adds the `attachment` type when mapping properties so that documents can be populated with file attachment contents (encoded as `base64`).
+
+Hello, world
```

Comment:
Could you move this section after `Installation` part? So it's looking more like what we have in our guide?

For example: https://www.elastic.co/guide/en/elasticsearch/plugins/2.0/analysis-icu.html